lib/pedicel/base.rb

```diff
@@ -43,40 +47,30 @@ def application_data
       [@token['applicationData']].pack('H*')
     end
 
-    def signing_time_ok?(now: Time.now)
-      # "Inspect the CMS signing time of the signature, as defined by section 11.3
-      # of RFC 5652. If the time signature and the transaction time differ by more
-      # than a few minutes, it's possible that the token is a replay attack."
-      # https://developer.apple.com/library/content/documentation/PassKit/Reference/PaymentTokenJSON/PaymentTokenJSON.html
-
-      delta = Pedicel.config[:replay_threshold_seconds]
-
-      cms_signing_time.between?(now - delta, now + delta)
-      # Deliberately ignoring leap seconds.
-    end
```

Comment:
Why should `signing_time_ok?` disappear?

Reply:
Hmm. I guess the predicate `signing_time_ok?` would be a good choice to keep, but as it is, the functionality existed in `self.verify_signed_time`.

Reply:
I see. And agree.

I think I tried making the replay-check part of the validation but gave up. (Notice `cms_signing_time` is left without an implementation 🙂)